pkg/identity/identity.go

```diff
@@ -527,3 +528,41 @@ func backupPath(path string) string {
 		pathExt,
 	)
 }
+
+// EncodePeerIdentity encodes the complete idenitity chain to bytes
```

Comment:
nit - spelling of identity

Reply:
Extremely sorry overlooked the comment and checked ... will  fix it ..